### ISSUE TITLE
Automatically exclude non-working files in bcc mode

### ISSUE THREAD

**Issue by ibot3** [dashingsoft/pyarmor#1512]
I am using pyarmor with `--enable-rft --enable-bcc ` options.
For a lot of files (around 10% of all python files), I get the following error:

```
INFO     obfuscating file __init__.py
DEBUG    process inline marker
DEBUG    parse script
INFO     start to generate bcc code ...
DEBUG    invalid co for bcc code caused by <frozen api.v1.notification>:26:pyarmor__2160
DEBUG    unknown error, please check pyarmor.error.log
ERROR    invalid co for bcc code
```

I excluded these files one by one as written in the documentation.
But I already have more than 10 exclusion rules and many more files to go and it takes a lot of time to do this try-error-exclude loop that many times.

Would it be possible to add an option so that files where the bcc mode fails are automatically excluded from compilation?

**Comment by jondy** [comment]
Got it, I'll try to find one solution.

**Comment by jondy** [comment]
Fixed in 8.3.10

If there is invalid bcc co, it will be ignored by bcc mode.

No `bcc:excludes` required.

**Comment by ibot3** [comment]
Thanks, I will test it tomorrow.
